airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -15,15 +15,33 @@
 
 from .streams import Annotations, CohortMembers, Cohorts, Engage, Export, Funnels, FunnelsList, Revenue
 from .testing import adapt_streams_if_testing, adapt_validate_if_testing
+from .utils import read_full_refresh
 
 
 class TokenAuthenticatorBase64(TokenAuthenticator):
```

Comment:
Is there any chance to use `BasicHttpAuthenticator` from CDK? 

`from airbyte_cdk.sources.streams.http.requests_native_auth.token import BasicHttpAuthenticator`

It could be called directly like this: 
```
BasicHttpAuthenticator(config["username"], config["secret"])
```